### ISSUE TITLE
DOI and year of publication

### ISSUE THREAD

**Issue by jafonso80** [sysrev/srvc-docs#1]
Hi!

Would it be possible to have SysRev capturing the DOI and year of publication? When importing the references to the platform via Zotero this data is usually available, and it would be useful when wanting to cross-check articles between projects for example.

**Comment by tomlue** [comment]
Yes, all citation data will be available in srvcs built on ris data. This issue will serve as a good reminder to make sure that works.
